src/components/Markings/components/ShapeEditorSvg.js

```diff
@@ -38,14 +54,17 @@ class ShapeEditorSvg extends Component {
             cornerLocations: {},
             closeLocations: {},
             shapeIndex: -1,
+            shapeToRemoveIndex: -1,
             touchState: {
                 bottomLeft: false,
                 bottomRight: false,
                 upperLeft: false,
                 upperRight: false,
                 onlySquare: false
             },
-            shapeRefs: {}
+            shapeRefs: {},
+            isDrawing: false,
+            previewShapeDimensions: this.previewShapeInitialDimensions()
```

Comment:
This seems odd. Why would `this.state.previewShapeDimensions` ever be needed as opposed to simply using `this.previewShapeInitialDimensions()` (with a different name, if needed)

Reply:
`previewShapeDimensions` gets modified in other places in the code, we only use `previewShapeInitialDimensions` to set the initial value. The reason we use a function for it is because it is based on some props passed in and those props may change during the lifecycle of the component.

Reply:
Gotcha. Saw it is also used when resetting items, but that makes sense.